test/integ/pandas/test_pandas_tools.py

```diff
@@ -111,6 +115,55 @@ def test_write_pandas(
             cnx.execute_string(drop_sql)
 
 
+@pytest.mark.parametrize(
+    "kwargs", [{"quote_identifiers": False}, {"quote_identifiers": True}]
+)
+def test_make_pd_writer(db_parameters: Dict[str, str], conn_cnx, kwargs: Dict):
+    with conn_cnx(
+        user=db_parameters["user"],
+        account=db_parameters["account"],
+        password=db_parameters["password"],
+    ) as cnx:  # type: SnowflakeConnection
```

Comment:
I see, I'll have another look - wasn't totally familiar with the use of `conn_cnx` when I wrote the test.

Reply:
I'll take your suggestion for the code, but I believe I'll need `db_parameters` since cnx won't let the `password` persist, and I require it when creating the sqlalchemy engine. Please let me know if there's a way around this.